src/Microsoft.VisualBasic/src/project.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
Correct me if I'm wrong, but shouldn't there be a change on the .vbproj here to remove project.json and add a `<Reference>` to System.Runtime?